sunpy/coordinates/tests/test_metaframes.py

```diff
@@ -110,16 +110,16 @@ def test_no_base():
         RotatedSunFrame()
 
 
-def test_no_sunpy_frame():
-    with pytest.raises(TypeError):
-        RotatedSunFrame(base=BaseCoordinateFrame)
-
-
 def test_no_obstime():
     with pytest.raises(ValueError):
         RotatedSunFrame(base=f.HeliographicStonyhurst(obstime=None))
 
 
+def test_non_sunpy_frame():
+    r = RotatedSunFrame(base=HeliocentricMeanEcliptic(obstime='2001-01-01'))
+    assert isinstance(r, RotatedSunFrame)
+
+
```

Comment:
```suggestion
```

Reply:
On second thought, let's put a non-SunPy frame through the exact same test for SunPy frames.  No reason not to.  Below the `rot_hpc` fixture above, add the following:
```python
# Test a non-SunPy frame
@pytest.fixture
def rot_hme():
    return (HeliocentricMeanEcliptic,
            RotatedSunFrame(lon=1*u.deg, lat=2*u.deg, distance=3*u.AU,
                            base=HeliocentricMeanEcliptic(obstime='2001-01-01', equinox='2001-01-01'),
                            duration=4*u.day))
```
And then add `"rot_hme"` to the list of fixtures before `test_class_creation()`.

Reply:
I had a conflict while pushing this changes to branch. Please, have a look at my commits, whether result is acceptable..